### PR TITLE
XTree: Init stub for converged tree DDS

### PR DESCRIPTION
**Pull request by DLehenbauer**
Initializing a stub DDS under /experimental/ as a workspace for collaborating on the converged SharedTree DDS and Property Tree DDS.

### CODE REVIEW

**Review by ksbrar — APPROVED** [review]
🚀

File: experimental/dds/xtree/src/packageVersion.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ *
+ * THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY
+ */
+
+export const pkgName = "@fluidexperimental/xtree";
```
Comment (tylerbutler) [inline experimental/dds/xtree/src/packageVersion.ts:8]:
```suggestion
export const pkgName = "@fluid-experimental/xtree";
```

**Review by tylerbutler — COMMENTED** [review]
I didn't look at the code, just focused on the infra/package stuff.

**Comment by DLehenbauer** [comment]
(@tylerbutler - Thanks for fixing pkgName.)

**Pull request merged**